crates/holochain/src/conductor/conductor.rs

```diff
@@ -732,37 +738,41 @@ where
     /// Associate a Cell with an existing App
     pub(super) async fn add_clone_cell_to_app(
         &mut self,
-        app_id: &InstalledAppId,
-        slot_id: &SlotId,
+        app_id: InstalledAppId,
+        slot_id: SlotId,
         properties: YamlProperties,
     ) -> ConductorResult<CellId> {
         let dna_store = &self.dna_store;
-        let (_, child_dna) = self
-            .update_state_prime(move |mut state| {
-                if let Some(app) = state.installed_apps_mut().get_mut(app_id) {
-                    let slot = app
-                        .slots()
-                        .get(slot_id)
-                        .ok_or_else(|| AppError::SlotIdMissing(slot_id.to_owned()))?;
-                    let parent_dna_hash = slot.dna_hash();
-                    let dna = dna_store
-                        .get(parent_dna_hash)
-                        .ok_or_else(|| DnaError::DnaMissing(parent_dna_hash.to_owned()))?
-                        .modify_phenotype(random_uid(), properties)?;
-                    Ok((state, dna))
-                } else {
-                    Err(ConductorError::AppNotRunning(app_id.clone()))
+        let (_, parent_dna_hash) = self
+            .update_state_prime({
+                let app_id = app_id.clone();
+                let slot_id = app_id.clone();
+                move |mut state| {
+                    if let Some(app) = state.installed_apps_mut().get_mut(&app_id) {
+                        let slot = app
+                            .slots()
+                            .get(&slot_id)
+                            .ok_or_else(|| AppError::SlotIdMissing(slot_id.to_owned()))?;
+                        let parent_dna_hash = slot.dna_hash().clone();
+                        Ok((state, parent_dna_hash))
+                    } else {
+                        Err(ConductorError::AppNotRunning(app_id.clone()))
+                    }
                 }
```

Comment:
Weird, is any state even being modified here? Doesn't look like it in the old or the new code. Maybe the update state can go away?